### PR TITLE
create global for MAX_ATTEMPTS to simplify debugging

### PR DESCRIPTION
**Pull request by pbugni**
In general, we use 3 attempts from front-end requests, should there be a blip that prevents an API request from responding.

When debugging code, this can be a hassle, as multiple threads concurrently hit breakpoints confusing the user.

Adding an obvious constant to make temporary changes to the number of attempts easy to adjust for dev purposes.

### CODE REVIEW

File: portal/static/js/src/modules/TnthAjax.js [hunk 1]
```diff
@@ -1133,7 +1134,7 @@ export default { /*global $ */
             return false;
         }
         params = params || {};
-        this.sendRequest("/api/user/" + userId + "/questionnaire_bank", "GET", userId, {data: {as_of_date: completionDate}, sync: params.sync}, function(data) {
+        this.sendRequest("/api/user/" + userId + "/questionnaire_bank", "GET", userId, {data: {as_of_date: completionDate}, sync: params.sync, max_attempts: MAX_ATTEMPTS}, function(data) {
```
Comment (pbugni) [inline portal/static/js/src/modules/TnthAjax.js:1137]: @achen2401 is this change necessary?  Wouldn't we pick up the correct `default` without including, as defined on line #21?

Comment (achen2401) [reply portal/static/js/src/modules/TnthAjax.js:1137]: @pbugni You are right, I was just thinking that. I think it is safe to remove the change here.

**Review by achen2401 — APPROVED** [review]
